src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -119,7 +126,7 @@ protected function import(RouteCollectionBuilder $routes, $confDir, $pattern)
 
         foreach ($configFiles as $resource) {
             if (!in_array($resource, $excludedConfigFiles)) {
-                $routes->import($resource, '/');
```

Comment:
Why is that slash not necessary anymore?

Reply:
Its the prefix which is by default `/` on the `RouteCollectionBuilder` https://github.com/symfony/symfony/blob/43688482bc01c40ec421ff9552e1455df460d3f3/src/Symfony/Component/Routing/RouteCollectionBuilder.php#L59

The new RouteConfiguration does not longer have this prefix: https://github.com/symfony/symfony/blob/43688482bc01c40ec421ff9552e1455df460d3f3/src/Symfony/Component/Routing/Loader/Configurator/RoutingConfigurator.php#L39

and for